Zonos/amino
# Refactor/css modules

## Jira issue
[FE-739 - Convert Styled Components to Css Modules: A-B](https://myzonos.atlassian.net/browse/FE-739)

## Description
This PR converts the following components to CSS Modules:
- AlertDialog
- AnnouncementDialog
- Avatar
- Badge
- Banner
- Button
  - MenuButton
  - Ripple
  - DropdownAnimation

## Testing
Domain: [css-modules.amino.zonos.com](https://css-modules.amino.zonos.com/)

1. Go to each of the components' stories listed above
2. Ensure that the styles and functionality work correctly
3. Change the variables to ensure that they are all working correctly

NOTE: Please double check the types in the `Button` component on lines 60-80 and the component return at line 306. These needed to be altered to allow for the mutable `tag` prop to be used. I got it to work, and I believe it's totally type safe, but I am inexperienced in this area.

## Todo

- [ ] Bump version and add tag
